etc/avocado/avocado.conf

```diff
@@ -19,6 +19,8 @@ enabled = True
 installed_packages = False
 # Whether to run certain commands in bg to give extra job debug information
 profiler = False
+# Force LANG for sysinfo collection
+lang = C
```

Comment:
Well I remember LC_ALL used to be not recommended back then when I was struggling with Czech locale as it forces really really everything to that language, so I used LANG, which changes the whole localization (including the number, date, ... format). But now when you pointed it out it's true that we do really want to always force everything to the defined localization, so let me fix that.